mopro-ffi/src/app_config/utils.rs

```diff
@@ -0,0 +1,67 @@
+use std::path::{Path, PathBuf};
```

Comment:
I'm not a fan of "utils" files. They generally end up being large files with poorly separated logic and a place for devs to put things they don't want to think about. The `mod.rs` file is only 73 lines, i think the contents of this file should remain there or this file should be named more intentionally based on it's contents.

Reply:
Well, that was my intention of throwing all extra logic in this file. I will think if there is a better name that can classify the functions in it. Perhaps I will put some back into `mod.rs`

Reply:
Changed to `sys_commands.rs`, and moved all the logic with tmp directories to `mod.rs`